src/main/java/core/basesyntax/FruitTransaction.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class FruitTransaction {
```

Comment:
Don't begin class implementation with an empty line. Please remove the redundant empty line at the start of the class implementation.